app/helpers/nodes_helper.rb

```diff
@@ -0,0 +1,26 @@
+module NodesHelper
+	# Gets the parents of a node,
+	# starting from root, ending at the node's direct parent
+  def parents(node)
+  	parents = []
+  	p = node.parent
+  	while p do
+  		parents.unshift(p)
+  		p = p.parent
+  	end
+  	parents
+  end
+
```

Comment:
This method should be part of the model, as it is providing information specifically related with the model. 
In the helpers is better to keep the functions that solve issues related with modifying/accessing data to be able to help rendering the view (like the rest of functions of this file).